axelrod/random_.py

```diff
@@ -3,6 +3,12 @@
 from axelrod import Actions
 
 
+choices = {
```

Comment:
Don't really have a preference. I tend to use dicts out of habit and I prefer either to conditionals. Not entirely sure why, though.....